akka-cluster-sharding/src/test/scala/akka/cluster/sharding/ClusterShardingLeaseSpec.scala

```diff
@@ -124,7 +124,7 @@ class ClusterShardingLeaseSpec(config: Config, rememberEntities: Boolean)
       testLease.setNextAcquireResult(Future.successful(true))
       expectMsg(3)
     }
-    "recover if lease lost" in {
+    "recover if lease lost" taggedAs GHExcludeTest in {
```

Comment:
Did we see this be unstable on GHA? https://github.com/akka/akka/issues/30455 reports it on jenkins (where it only occurs rarely)

Reply:
removed